### PR TITLE
V1.7.0

### PR DESCRIPTION
**Pull request by IvanPleshkov**
QDrant API v1.7.0 integration

Also I merged to this branch this change from user:
https://github.com/qdrant/qdrant-js/pull/54
And updated dependencies for proto generator (SNYK recomendation) and for undici (because of node packages audit)

### CODE REVIEW

File: packages/js-client-rest/src/qdrant-client.ts [hunk 1]
```diff
@@ -415,16 +463,18 @@ export class QdrantClient {
...
+     *     - shard_key: Specify in which shards to look for the points, if not specified - look in all shards
+     *     - filter: filtering conditions
      *     - exact:
      *         If `True` - provide the exact count of points matching the filter.
      *         If `False` - provide the approximate count of points matching the filter. Works faster.
      *         Default: `true`
      * @returns Amount of points in the collection matching the filter.
      */
-    async count(collection_name: string, {filter, exact = true}: SchemaFor<'CountRequest'> = {}) {
+    async count(collection_name: string, {shard_key, filter, exact = true}: SchemaFor<'CountRequest'> = {}) {
```
Comment (IvanPleshkov) [inline packages/js-client-rest/src/qdrant-client.ts:474]: Check if read_consistency is required. Grpc introduced `read_consistency` field in count request

Comment (IvanPleshkov) [reply packages/js-client-rest/src/qdrant-client.ts:474]: It's strange  for me, but count request does not have this parameter in openapi

File: packages/js-client-grpc/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # @qdrant/js-client-grpc
 
+## 1.7.0
```
Comment (IvanPleshkov) [inline packages/js-client-grpc/CHANGELOG.md:3]: Add package updates

**Review by timvisee — APPROVED** [review]

File: packages/js-client-grpc/CHANGELOG.md (hunk 2 above)
Comment (IvanPleshkov) [reply packages/js-client-grpc/CHANGELOG.md:3]: Added thanks to @Morad-m11 for `recommendBatch` function

**Pull request merged**